tools/DfmHttpService/Model/CommandMessage.cs

```diff
@@ -19,16 +19,19 @@ internal class CommandMessage
         [JsonProperty("markdownContent")]
         public string MarkdownContent { get; set; }
 
-        [JsonProperty("writeTempPreviewFile")]
-        public bool WriteTempPreviewFile { get; set; }
+        [JsonProperty("ifWriteTempPreviewFile")]
+        public bool IfWriteTempPreviewFile { get; set; }
```

Comment:
remove this parameter.

---
In reply to: [113350507](https://github.com/dotnet/docfx/pull/1590#discussion_r113350507) [](ancestors = 113350507)